### PR TITLE
Homework 12. Sections What clients say and Meet our team.

### PR DESCRIPTION
**Pull request by VioletaScvortova**
Hello! I've added below 2 screenshots with my homework. Sections What clients say and Meet our team. 
There was some kind of error and the same commit "Added What Clients Say content" was created twice.
Looking forward to your reviews))

Lesson 12_ Homework  Page What clients say
Lesson 12_ Homework  Page Meet our team

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
...
+            <div class="feedback__content">
+                <h2 class="content__general-title">What Clients Say</h2>
+                <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
+                    Classical physics: Newtonian mechanics<p>
             </div>
 
+            <div class="grid-container">
+                <div class="grid-card" id="card-items">
+                    <div class="stars">
+                        <span data-rate="1"></span>
```
Comment (Irrreny) [inline index.html:133]: I think you could use simple images here and add only width and height in css, so in this way you could have saved some code. But i guess it works both ways :)

File: index.html [hunk 2]
```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
...
+            </div>
 
+            <div class="grid-container">
+                <div class="grid-card" id="card-staff">
+                    <div class="staff-info">
+                        <img src="./img/staff-ashley-fletcher.png" alt="Ashley Fletcher photo" class="staff__photo">
+                        <div>
+                            <div class="staff__content">
+                                <h5 class="staff__name">Ashley Fletcher</h5>
+                                <h6 class="staff__job">CEO</h6>
```
Comment (Irrreny) [inline index.html:214]: You can style capital letters in css.

**Review by Irrreny — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (VioletaScvortova) [reply index.html:214]: Oh, yees, exactly, thank you! I forgot about the possibility - "text-transform: uppercase;"

File: index.html (hunk 1 above)
Comment (VioletaScvortova) [reply index.html:133]: I`ve added url with images (svg) to .stars and .empty-stars classes. I found this way to display the rating and I also have a question if it`s ok to use this way and to bother with it))

Thank you a lot for your review!

File: index.html [hunk 3]
```diff
@@ -66,10 +66,11 @@ <h4 class="content__mission">We know how large objects will act, but things on a
 
     <section class="strong-points">
         <div class="container">
+
             <div class="strong-points__content">
                 <h2 class="content__title">Why choose us</h2>
                 <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
```
Comment (LiluUncharted) [inline index.html:72]: "br" not the best option to use, it's better to set styles in css, just assign a max-width and align to the center )

**Review by LiluUncharted — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (VioletaScvortova) [reply index.html:72]: I'll keep this in mind for the future, thank you for your advice!

File: index.html [hunk 4]
```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
...
+                                    <img src="./img/svg/instagram-icon.svg" alt="Instagram icon icon"
+                                        class="social-media">
+                                    <img src="./img/svg/twitter-icon.svg" alt="Twitter icon" class="social-media">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="grid-card" id="card-staff">
```
Comment (IvanCeban) [inline index.html:244]: id should be unique.
You use `id="card-staff"` several times, which is forbidden

Comment (IvanCeban) [reply index.html:244]: it's not only about card-staff. Check all your ids

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Very well done!
Please check the comments

**Comment by VioletaScvortova** [comment]
@IvanCeban hello! I've changed the rating format and added a section with the company video presentation. Please, check if everything is well done. 

As for the gradient using png, I searched on google and double-checked a lot, but I still couldn’t find the reason why this option didn’t work.

Lesson 12_ Homework  Page Video presentation

File: index.html (hunk 4 above)
Comment (VioletaScvortova) [reply index.html:241]: done!

Comment (VioletaScvortova) [reply index.html:241]: @IvanCeban thanks a lot for the detailed explanation! Now it`s clear

**Pull request merged**